exploration/hist_monitor_plotting.md

```diff
@@ -0,0 +1,357 @@
+---
+jupyter:
+  jupytext:
+    formats: ipynb,md
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.16.1
+  kernelspec:
+    display_name: ds-aa-cub-hurricanes
+    language: python
+    name: ds-aa-cub-hurricanes
+---
+
+# Historical monitor plotting
+<!-- markdownlint-disable MD013 -->
+Plotting historical triggers based on historical forecasts
+
+```python
+%load_ext jupyter_black
+%load_ext autoreload
+%autoreload 2
+```
+
+```python
+import ocha_stratus as stratus
+import pandas as pd
+import matplotlib.pyplot as plt
+import matplotlib.patches as mpatches
+
+from src.constants import *
+```
+
+## Load and merge data
+
+```python
+# temporarily load IBTrACS locally since DB is down
+filename = "temp/ibtracs.NA.list.v04r01.csv"
+df_ibtracs = pd.read_csv(filename, skiprows=[1])
+```
+
+```python
+df_storms = (
+    df_ibtracs.groupby("SID")[["NAME", "USA_ATCF_ID"]].last().reset_index()
+)
+df_storms = df_storms.rename(
+    columns={"SID": "sid", "NAME": "name", "USA_ATCF_ID": "atcf_id"}
+)
+df_storms = df_storms[df_storms["sid"].str[:4].astype(int) >= 2000]
+df_storms["atcf_id"] = df_storms["atcf_id"].str.lower()
+```
```

Comment:
perhaps you can replace temp file w/ db connection for long-term reproducibility. I replaced the above with:

```
from src.datasources import ibtracs

df_storms = ibtracs.load_storms()
````

and it seemed to be fine